js/scripts.js

```diff
@@ -0,0 +1,7 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+var elephant = "Zielone słonie";
+var bigGreenElephant = elephant.toUpperCase();
+var animal = text.replace('Papugi', bigGreenElephant);
```

Comment:
zrób tak żeby zamiast Papug, zawsze był zastępowany pierwszy wyraz z `text` - bez względu na to co tam będzie. Możesz wykorzystać do tego metodę `split` (google -> MDN split)